.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   build:
     uses: Datadog/action-prebuildify/.github/workflows/build.yml@main
     with:
-      package-manager: 'npm'
+      package-manager: 'yarn'
```

Comment:
The lockfiles need to be moved over to yarn too.

Reply:
Done, good catch.